lib/commands/screenshots.js

```diff
@@ -66,6 +78,19 @@ commands.getScreenshot = async function () {
     return data;
   };
 
+  // ensure the user doesn't try to use 2 specialty screenshot caps
+  if (this.opts.realDeviceScreenshotter && this.mjpegStream) {
+    log.warn("You've specified screenshot retrieval via both MJpeg server " +
+             "and a real device screenshot utility. Please use one or the " +
+             "other! Choosing MJpeg server");
+  }
+
+  // if we've specified an mjpeg server, use that
+  if (this.mjpegStrem) {
+    return await getScreenshotFromMJpeg();
```

Comment:
we should verify the returned result here otherwise the fallback won't work the way you expect it to work.

Reply:
good point.